### PR TITLE
chore: give the design-system website a little love

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* Updates deps
* Fixes poor implementation of styled-components in SSR

### Why is it needed?

* Stops FOUC
* Keeps the site up to date

### CODE REVIEW

File: website/pages/_document.js [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+          <>
+            {initialProps.styles}
+            {sheet.getStyleElement()}
+          </>
+        ),
+      };
+    } finally {
+      sheet.seal();
+    }
   }
```
Comment (gu-stav) [inline website/pages/_document.js:26]: Do we need that at all? Looking at https://dev.to/britotiagos/how-add-style-components-to-nextjs-and-start-using-it-4kdf for example it looks like as if we could solve this using a babel-plugin ... I think with next@13 this became an swc plugin ...

**Review by ronronscelestes — COMMENTED** [review]
love the title
a small question - other than that looks good, thank you!

File: website/components/Layout/index.js [hunk 2]
```diff
@@ -14,14 +13,12 @@ const Layout = ({ children, meta, navigationContent }) => {
         <Head {...meta} />
         <Box padding={4}>
           <ContentLayout>
-            <Main>
```
[the remark this answers is not in the document]
Comment (ronronscelestes) [reply website/components/Layout/index.js:17]: makes sense,
I was thinking about removing as="main" for Stack instead:
```js
<Main>
  <Stack>{children}</Stack>
</Main>
```

but as you prefer :)

[the remark this answers is not in the document]
Comment (ronronscelestes) [reply website/components/Layout/index.js:17]: ah my bad! thanks 😊

**Review by ronronscelestes — APPROVED** [review]
LGTM! 🧹

File: website/pages/_document.js (hunk 1 above)
Comment (gu-stav) [reply website/pages/_document.js:26]: Hell. Thanks!

**Pull request merged**